tests/data/test_solution.archive.yaml

```diff
@@ -1,2 +1,2 @@
 data:
-  m_def: nomad_material_processing.solution.general.Solution
\ No newline at end of file
+  m_def: nomad_material_processing.solution.Solution
```

Comment:
I changes this for testing the alliases. Maybe you want to change this back or have two separate test cases.